src/models/todos.js

```diff
@@ -1,88 +1,73 @@
 import Cycle from 'cyclejs';
+import {Map} from 'immutable';
```

Comment:
Not sure if it's good practice to have `Map` conflicting with the native one in new JS engines ...